src/lib/datepicker/datepicker.md

```diff
@@ -51,6 +51,23 @@ year containing the `startAt` date.
 
 <!-- example(datepicker-start-view) -->
 
+#### Watching the views for changes on selected years and months
+
+When a year or a month is selected in **multi-year and year views** respecively, the `yearSelected`
+and `monthSelected` outputs emit the chosen year (full year) and month (0 - 11).
```

Comment:
now that I think about this some more, wouldn't it make sense to emit normalized dates
(e.g. `new Date(year, 0, 1)` and `new Date(year, month, 1)`)? especially for the month case, the month number alone doesn't give you all the info you need

Reply:
ok, I'm gonna update this and push it tonight.